app/bibleview-js/src/utils.ts

```diff
@@ -616,6 +616,6 @@ export function sprintf(format: string, ...args: any[]) {
 }
 
 export function formatExportLink({ref, v11n, doc}: {ref: string, v11n: string, doc?: string}) {
-    const docStr = doc ? `&document=${doc}`: ""
-    return `https://stepbible.org/?q=reference=${ref}&v11n=${v11n}${docStr}`
-}
\ No newline at end of file
+    const docStr = doc ? `|version=${doc}`: ""
+    return `https://stepbible.org/?q=reference=${ref}${docStr}&v11n=${v11n}`
+}
```

Comment:
This change is not enough: we need to also intepret this formatting correctly in MainBibleActivity:openLink

Reply:
(in case if user clicks link on a device that has andbible set to handle stepbible.org links)

Reply:
I think this is fixed now, but in debug something seems problematic. It seems to start a new AB activity each time and then only opens the reference occasionally. I haven't seen this issue in release build from before though.